contracts/utils/contracts/src/Refundable.sol

```diff
@@ -22,32 +22,58 @@ pragma solidity ^0.5.9;
 contract Refundable {
 
     // This bool is used by the refund modifier to allow for lazily evaluated refunds.
-    bool internal shouldNotRefund;
+    bool internal _shouldNotRefund;
 
     modifier refundFinalBalance {
         _;
-        if (!shouldNotRefund) {
-            _refundNonzeroBalance();
-        }
+        _refundNonZeroBalanceIfEnabled();
     }
 
     modifier disableRefundUntilEnd {
-        if (shouldNotRefund) {
+        if (_areRefundsDisabled()) {
             _;
         } else {
-            shouldNotRefund = true;
+            _disableRefund();
             _;
-            shouldNotRefund = false;
-            _refundNonzeroBalance();
+            _enableRefund();
+            _refundNonZeroBalance();
         }
     }
 
-    function _refundNonzeroBalance()
+    function _refundNonZeroBalanceIfEnabled()
+        internal
+    {
+        if (!_areRefundsDisabled()) {
```

Comment:
Nit: Is there an issue with inlining this function? It's really short already, so we could save the `JUMPs` and `JUMPDESTs` to and from by removing the `_areRefundsDisabled` function.

Reply:
Removing this adds 107 bytes of codesize, which pushes us off the edge.

Reply:
Wow, that's crazy. I'm going to have to look into that 🤔

Reply:
As mentioned offline, was the addition of 107 bytes a result of removing `_refundNonZeroBalanceIfEnabled` or `_areRefundsDisabled`? The first would make sense, but the latter really wouldn't.

Reply:
It was from removing `_areRefundsDisabled` haha